bigquery_etl/cli/utils.py

```diff
@@ -250,3 +252,25 @@ def temp_dataset_option(
         help="Dataset where intermediate query results will be temporarily stored, "
         "formatted as PROJECT_ID.DATASET_ID",
     )
+
+
+def get_glean_channel_to_app_name_mapping() -> Dict[str, str]:
```

Comment:
I recommend renaming this to something more general now that it's used in more places
```suggestion
def get_glean_app_id_to_app_name_mapping() -> Dict[str, str]:
```